tests/007-test-setting-extra-opts.sh

```diff
@@ -45,7 +45,11 @@ EOF
   echo $DOCKER_STORAGE_OPTIONS | grep -q -- "$extra_options"
   
   # Successful appending to DOCKER_STORAGE_OPTIONS
-  [ $? -eq 0 ] && test_status=0
+  if [ $? -eq 0 ]; then
+      test_status=0
+  else
+      echo "ERROR: $testname: $extra_options != $DOCKER_STORAGE_OPTIONS." >> $LOGS
```

Comment:
We are doing grep. So DOCKER_STORAGE_OPTIONS should contain extra_options and does not have to be same.

Can we change error message to reflect that. Maybe.

` ERROR: $testname: DOCKER_STORAGE_OPTIONS does not contain extra_options`